perf/benchmark/run_benchmark_job.sh

```diff
@@ -75,29 +86,58 @@ pushd "${WD}"
 ./setup_test.sh
 popd
 
+# Setup fortio and prometheus
 setup_metrics
+
 echo "Start running perf benchmark test."
 # For adding or modifying configurations, refer to perf/benchmark/README.md
-# Configuration1
 EXTRA_ARGS="--serversidecar --baseline"
+# Configuration Set1: CPU and memory with mixer enabled
+MIXER_MODE="--mixer_mode mixer"
 CONN=16
-QPS=500,1000,1500,2000
-DURATION=300
-METRIC="cpu"
-# shellcheck disable=SC2086
-pipenv run python3 runner.py ${CONN} ${QPS} ${DURATION} ${EXTRA_ARGS}
-collect_metrics true ${METRIC}
-METRIC="mem"
-collect_metrics true ${METRIC}
-
-# Configuration2
+QPS=10,100,500,1000,2000,3000
+DURATION=240
+METRICS=(cpu mem)
+get_benchmark_data
+
+# Configuration Set2: Latency Quantiles with mixer enabled
+CONN=1,2,4,8,16,32,64
+QPS=1000
+METRICS=(p50 p90 p99)
+get_benchmark_data
+
+# Configuration Set3: CPU and memory with mixer disabled
+kubectl -n istio-system get cm istio -o yaml > /tmp/meshconfig.yaml
+pipenv run python3 ./update_mesh_config.py disable_mixer /tmp/meshconfig.yaml | kubectl -n istio-system apply -f /tmp/meshconfig.yaml
+MIXER_MODE="--mixer_mode nomixer"
+CONN=16
+QPS=10,100,500,1000,2000,3000
+DURATION=240
+METRICS=(cpu mem)
+get_benchmark_data
+
+# Configuration Set4: Latency Quantiles with mixer disabled
+CONN=1,2,4,8,16,32,64
+QPS=1000
+METRICS=(p50 p90 p99)
+get_benchmark_data
+
+# Configuration Set5: CPU and memory with mixer v2
```

Comment:
instead of mixerv2, use `prometheusv2-nullvm`   
later on we will have `prometheusv2-v8` and prometheusv2-wavm`

Reply:
done